pkg/kubelet/apis/cri/v1alpha1/runtime/api.pb.go

```diff
@@ -1407,10 +1416,17 @@ type LinuxContainerSecurityContext struct {
 	// UID to run the container process as. Only one of run_as_user and
 	// run_as_username can be specified at a time.
 	RunAsUser *Int64Value `protobuf:"bytes,5,opt,name=run_as_user,json=runAsUser" json:"run_as_user,omitempty"`
+	// GID to run the container process as. Only one of run_as_group and
+	// run_as_groupname can be specified at a time.
+	RunAsGroup *Int64Value `protobuf:"bytes,12,opt,name=run_as_group,json=runAsGroup" json:"run_as_group,omitempty"`
 	// User name to run the container process as. If specified, the user MUST
 	// exist in the container image (i.e. in the /etc/passwd inside the image),
 	// and be resolved there by the runtime; otherwise, the runtime MUST error.
 	RunAsUsername string `protobuf:"bytes,6,opt,name=run_as_username,json=runAsUsername,proto3" json:"run_as_username,omitempty"`
+	// Group name to run the container process as. If specified, the group MUST
+	// exist in the container image (i.e. in the /etc/group inside the image),
+	// and be resolved there by the runtime; otherwise, the runtime MUST error.
+	RunAsGroupname string `protobuf:"bytes,13,opt,name=run_as_groupname,json=runAsGroupname,proto3" json:"run_as_groupname,omitempty"`
```

Comment:
@pmorie see above there is a parallel called `RunAsUsername` field. Do you suggest i remove this ? i think this would allow similar verification in /etc/group but if you feel this is not needed , i can remove it

Reply:
@pmorie  fixed